README_SYCL.md

```diff
@@ -102,3 +97,25 @@ These can be used interchangeably (some restrictions apply - see below) with the
   ```
   See [Intel's FAQ](https://github.com/intel/compute-runtime/blob/master/opencl/doc/FAQ.md#feature-double-precision-emulation-fp64) for more information.
 * The FPGA back-end does not support atomics. alpaka will not check this.
+* Device global variables (corresponding to `__device__` and `__constant__` variables in CUDA) are not supported in the SYCL back-end yet.
+* Shared memory does not yet work properly on CPU back-ends. The tests that use it (`atomicTest`, `blockSharedTest` and `blockSharedSharingTest`) fail with a `PI_ERROR_OUT_OF_RESOURCES`.
+
+### Choosing the sub-group size (warp size)
+
+Most SYCL targets support multiple sub-group sizes. There is a trait to specify at compile time the sub-group size to use for a kernel that can be defined as follow:
+```cpp
+template<std::uint32_t TWarpSize, typename TAcc>
+struct alpaka::trait::WarpSize<MyKernel<TWarpSize>, TAcc>
+    : std::integral_constant<std::uint32_t, TWarpSize>
+{
+};
+```
+`MyKernel` must therefore be templated on the sub-group size.
```

Comment:
```suggestion
For example, if `MyKernel` requires a sub-group size of `32`, this can be declared specialising the `alpaka::trait::WarpSize`:
```cpp
struct MyKernel { ... };

template<typename TAcc>
struct alpaka::trait::WarpSize<MyKernel, TAcc>
    : std::integral_constant<std::uint32_t, 32>
{
};
```
This can be extended to kernels that support multiple sub-group sizes at compile time:
```cpp
template<std::uint32_t TWarpSize>
struct MyKernel { ... };

template<std::uint32_t TWarpSize, typename TAcc>
struct alpaka::trait::WarpSize<MyKernel<TWarpSize>, TAcc>
    : std::integral_constant<std::uint32_t, TWarpSize>
{
};
```
```